### ISSUE TITLE
How about a $ghdog telegram ? Would you agree

### ISSUE THREAD

**Issue by Cryptobertl** [thisrepo/dog#39]
Let’s get the supporters together

**Comment by Tarryco** [comment]
> Let’s get the supporters together

No one had created a telegram yet, can you create one first?

**Comment by VegasBtc** [comment]
LFG

**Comment by Cryptobertl** [comment]
Sure we cannot we get enough people to agree here

**Comment by Cryptobertl** [comment]
I will go to dinner now with my lovely wife if no one created before me I will make one then .. post the link here if someone did it already an I would join as admin to help.

**Comment by zhouyaohua** [comment]
I just created a Telegram Group. Please join.

https://t.me/+jXS6UG2t5S4wZjNl

**Comment by Cryptobertl** [comment]
Awesome mate

**Comment by thisrepo** [comment]
Made an official update on that topic: https://github.com/thisrepo/dog/issues/42

**Comment by zhouyaohua** [comment]
Telegram: https://t.me/+jXS6UG2t5S4wZjNl

**Comment by Cryptobertl** [comment]
Can you rename the tg to dhdog

**Comment by Tarryco** [comment]
> dhdog

dhdog?

**Comment by Tarryco** [comment]
> Awesome mate

I have no experience with managing Telegram group, Could you please create a new Telegram Group?

**Comment by Cryptobertl** [comment]
Did you delete the telegram?

**Comment by Tarryco** [comment]
> Did you delete the telegram?

Yes, please create a new again, because i am not good at Telegram Group management.

But i can provide Java, Vue development, Ops deployment capabilities.

**Comment by Cryptobertl** [comment]
Ok I’ll make it here during the days actually it’s 6 o clock on Sunday and I want to get up nicely with my sexy wife now

**Comment by Tarryco** [comment]
> Ok I’ll make it here during the days actually it’s 6 o clock on Sunday and I want to get up nicely with my sexy wife now

If ghdog takes off, your sexy wife will love you even more.

**Comment by Cryptobertl** [comment]
Haha 😂

**Comment by Cryptobertl** [comment]
https://t.me/ghdogsol Not fully setup but I can mod up some of you and we can start please check if safeguard is working
